src/cljc/harja/domain/lupaus_domain.cljc

```diff
@@ -380,6 +380,20 @@
 (defn etsi-lupaus-kuukausi [kuukaudet kohdekuukausi]
   (first (filter #(= kohdekuukausi (:kuukausi %)) kuukaudet)))
 
+(defn etsi-nykyinen-valinta-askel [valittu-arvo vaihtoehdot]
+  (let [arvo (first (filter #(= (:id %) valittu-arvo) vaihtoehdot))
+        vaihtoehto-askel (:vaihtoehto-askel arvo)]
+    vaihtoehto-askel))
+
+(defn etsi-edeltavat-monivalinnan-valitut-arvot [valittu-arvo vaihtoehdot]
+  (let [arvo (first (filter #(= (:id %) valittu-arvo) vaihtoehdot))
```

Comment:
Ei oo pakko tehdä, mutta jaan ajatuksen silti.
Näitä tarpeita, jossa mäpistä otetaan jonku avaimen arvon perusteella ensimmäinen, on tosi paljon.
Ja jokainen rakentaa tämän saman filtteri first homman erikseen.
Niin mietin, että oisko kova, jos ois joku yhteinen funktio tähän, tyyliin

(defn listan-ensimmäinen-avaimen perusteella [avain valittava-arvo lista]
(first (filter #(= (keyword avain %) valittava-arvo) lista)))